ScoutSuite/frontend/src/pages/404/style.scss

```diff
@@ -0,0 +1,3 @@
+.page-404 {
+  
```

Comment:
was there supposed to be styles for the 404 page?

Reply:
Yes, I wanted to make it look nice!
![image](https://user-images.githubusercontent.com/1829117/114633364-604ed400-9c8e-11eb-8525-454c7907bd92.png)